include/csoundCore.h

```diff
@@ -1,7 +1,7 @@
 /*
-  csoundCore.h:
+  csoundCore.h: somet modifications to match the new API
```

Comment:
The file description here seems off with "somet" and just in general. Looks like and odd typo?